tfh/lib/tfh/cmd/tfh_run_apply.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+## -------------------------------------------------------------------
+##
+## Copyright (c) 2018 HashiCorp. All Rights Reserved.
+##
+## This file is provided to you under the Mozilla Public License
+## Version 2.0 (the "License"); you may not use this file
+## except in compliance with the License.  You may obtain
+## a copy of the License at
+##
+##   https://www.mozilla.org/en-US/MPL/2.0/
+##
+## Unless required by applicable law or agreed to in writing,
+## software distributed under the License is distributed on an
+## "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+## KIND, either express or implied.  See the License for the
+## specific language governing permissions and limitations
+## under the License.
+##
+## -------------------------------------------------------------------
+
+tfh_run_apply () {
+  run_id="$1"
+  comment="$2"
+
+  if ! check_required token address; then
```

Comment:
you can change this to just `if ! check_required; then` - it will default to checking for everything, which is needed for this function. No error message needed in the body, just return 1. the function handles the error message printing.